custom/covid/management/commands/fetch_form_case_counts.py

```diff
@@ -39,6 +39,7 @@ def get_rows(self, domains, case_types, start, end):
         for domain in _expand_domains(domains):
             submissions_counts = _get_submissions_counts(domain, start, end)
             case_update_counts = _get_case_update_counts(domain, start, end)
```

Comment:
oh, you ended up keeping this bit in there?  I would've thought it'd give you those timeout errors I was seeing.

Reply:
Ah that makes sense.  No strong feelings - fine to keep around, as there's a decent chance we'll keep tweaking this.  The downside with keeping it in is that we'll have to make a private release if we need to run it again, but we'd probably need further code tweaks anyways, so :shrug: